perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -90,6 +94,27 @@ def AptInstall(vm):
   _Install(vm)
 
 
+def JujuInstall(vm):
+  if vm.isController is True:
+    if vm.vm_group_specs and 'workers' in vm.vm_group_specs:
+      vm.JujuDeploy('cs:trusty/cassandra',
+                    vm.vm_group_specs['workers'].vm_count,
+                    'workers')
```

Comment:
I think I see what you're saying, about how other benchmarks may reuse the cassandra package (such as cassandra_ycsb).

So the above code would work to install both charms into the right groups. There'd need to be another logical step or two added, though, to run `vm.JujuRelate('cassandra', 'cassandra-stress')` and `vm.JujuWait()`. The relate could be called from the Install in the cassandra-stress package, since that package would require cassandra being installed, and `JujuWait()` could be installed in the package's StartCluster() method, instead of returning if os_types.JUJU.

Am I on the right track with that?